.github/ISSUE_TEMPLATE/ops_onboarding_checklist.md

```diff
@@ -58,17 +58,12 @@ High level:
 - [ ] [Delivery Infrastructure Overview demo presentation](https://docs.google.com/presentation/d/1dXE3kkG56fXu7QvqH7qu5mGE3gfalD0FRf-gbnnjnHc/edit?usp=sharing)
 - Meet the members of the team
   - [ ] Jesse
-  - [ ] Demian
   - [ ] Jeremy
   - [ ] Matt
-  - [ ] Bill
-  - [ ] Faith
-  - [ ] Sri
-  - [ ] Ryan
-  - [ ] Robert
-  - [ ] Gary
-  - [ ] Troy
-  - [ ] Azeez
+  - [ ] John
+  - [ ] Oseas
+
```

Comment:
Wow, we have slimmed down quite a bit. Make sure to include yourself!

Reply:
oops! done ✅